cmd/create/cluster/cmd.go

```diff
@@ -981,79 +981,80 @@ func run(cmd *cobra.Command, _ []string) {
 
 	// Billing Account
 	billingAccount := args.billingAccount
-	if billingAccount != "" && !isHostedCP {
-		r.Reporter.Errorf("Billing accounts are only supported for Hosted Control Plane clusters")
-		os.Exit(1)
-	}
-
-	if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
-		r.Reporter.Errorf("Billing account is invalid. Run the command again with a valid billing account.")
-		os.Exit(1)
-	}
+	if isHostedCP {
+		if billingAccount != "" {
+			r.Reporter.Errorf("Billing accounts are only supported for Hosted Control Plane clusters")
+			os.Exit(1)
+		}
```

Comment:
@chenz4027 can you please review this logic? It's currently not accepting billing accounts for hosted control plane clusters.

Reply:
I moved it to an else statement, thanks for catching the flaw: 
![image](https://github.com/openshift/rosa/assets/26282541/cce752c5-46be-47b8-9961-e2952313ba6d)